packages/components/app/styles/components/pagination.scss

```diff
@@ -69,15 +70,19 @@
   &.mock-active {
     color: var(--token-color-foreground-action-active);
   }
+}
 
+// only the "Previous" & "Next" (`ButtonArrow`) controls can be disabled
+.hds-pagination-nav__button-arrow {
+  // notice: when `@disabled={{true}}` is applied, the code always renders a `<button>` element
   &:disabled,
   &.mock-disabled {
     color: var(--token-color-foreground-disabled);
     cursor: not-allowed;
   }
 }
 
-// Make buttons with "Previous" & "Next" labels the same min-width
+// Make buttons with "Previous" & "Next" buttons the same min-width
 // to make them more visually centered in relationship to other sub-components
 .hds-pagination-nav__button-arrow--labeled {
   min-width: 103px;
```

Comment:
Reminder: this is one we need to discuss again with @cveigt before final release.

Reply:
I am going to remove it for simplicity, we can always add it later if we see it's needed.